test/test.cpp

```diff
@@ -8194,4 +8201,60 @@ TEST(Longtail, Longtail_CaseSensitivePaths)
     SAFE_DISPOSE_API(compression_registry);
     SAFE_DISPOSE_API(target_storage);
     SAFE_DISPOSE_API(source_storage);
+}
+
+TEST(Longtail, PlatformWriteLargeFile)
+{
+    static const uint64_t expected_size = 4ULL * 1024ULL * 1024ULL * 1024ULL;
```

Comment:
I don't think we should run these tests as default - creating a 4Gb file on each test run seems excessive.
Also, we could combine PlatformWriteLargeFile and PlatformReadLargeFile into one test so we don't have to generate the large file twice.

Reply:
Is there a way to express default/non-default in the testing framework or should I wrap them in `#if 0`?

Reply:
I don't think there is a non-default option. But just encapulate it inside a if (false) { } and it is easy to just change the flag.